src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs

```diff
@@ -36,7 +38,8 @@ public class PatientDiscovery
             if (await AlreadyExists(request.TransactionId))
             {
                 return (null,
-                    new ErrorRepresentation(new Error(ErrorCode.DuplicateDiscoveryRequest, "Discovery Request already exists")));
+                    new ErrorRepresentation(new Error(ErrorCode.DuplicateDiscoveryRequest,
+                        "Discovery Request already exists")));

```

Comment:
Is it needed?

Reply:
I remember you mentioning in another PR that line should not exceed some 80 columns

Reply:
120's the default column limit. I might have said 80/something whatever is default.

Reply:
it is at 120.